chia/cmds/sim.py

```diff
@@ -34,10 +35,9 @@
 )
 @click.pass_context
 def sim_cmd(ctx: click.Context, rpc_port: Optional[int], root_path: str, simulator_name: str) -> None:
-    ctx.ensure_object(dict)
-    ctx.obj["root_path"] = Path(root_path) / simulator_name
-    ctx.obj["sim_name"] = simulator_name
```

Comment:
This is an intentional deletion that does not get re-added in the class context?

Reply:
the `simulator_name` value from the cli is used for creating a sub-'root path'.  it's assignment into the context dict was unused and thus removed.